exp: return nan instead of crashing the interpreter (#430)
adrn/gala: CHANGES.rst

- Added the ability to specify integer or string (i.e. non-Quantity) potential
  parameters.

> - Added ``gala.potential.EXPPotential`` for using EXP as a source of potentials

Reviewer: ```suggestion
- Added ``gala.potential.EXPPotential`` for using basis function expansion potentials from EXP. 
```
